src/main/java/com/atlassian/tutorial/Service/AttachmentsServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.atlassian.tutorial.Service;
+
+import com.atlassian.activeobjects.external.ActiveObjects;
+import com.atlassian.plugin.spring.scanner.annotation.imports.ComponentImport;
+import com.atlassian.tutorial.entity.AttachmentsEntity;
+import net.java.ao.Query;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+@Named
+public class AttachmentsServiceImpl implements AttachmentsService {
+
+    private static final String SQL_QUERY = "PAGE_ID = ? AND USER_ID = ? ORDER BY ID";
```

Comment:
do you really need `ORDER BY` while you store just one attachment for page and user?